src/overlays/actors/ovl_En_Pst/z_en_pst.c

```diff
@@ -18,7 +18,78 @@ void EnPst_Draw(Actor* thisx, PlayState* play);
 void func_80B2BD98(EnPst* this, PlayState* play);
 void func_80B2BE54(EnPst* this, PlayState* play);
 
-#if 0
+typedef enum {
+    /* 0 */ POSTBOX_SCH_NONE,
+    /* 1 */ POSTBOX_SCH_AVAILABLE
+} PostboxScheduleResult;
```

Comment:
Just want to make a small note especially since the schedule stuff is so new. This enum should have the same values as what is returned by the scripts (which looks to be 0, 1, 2). They should also be used in the schedule scripts themselves, as well as wherever the result from the `ScheduleOuptut` gets compared.

Reply:
Are you fine with having 2 be `/* 2 */ POSTBOX_SCH_UNK`? From what I saw in KZ, if I set it to 0, it will briefly despawn, but 2 had no observable changes

Reply:
Based on the code below, 2 looks to just be `POSTBOX_SCH_DO_NOTHING`